encryption/README.md

```diff
@@ -0,0 +1,52 @@
+# Encryption
+
+## What is it?
+
+Encryption involves complex mathematics to obscure data so that it is almost impossible to read without the correct decryption keys or passwords. End-to-end encryption is commonly used when transferring information that needs to be kept secret over a public system such as the internet.
```

Comment:
Are we capitalising `internet` or not? Perhaps we need a Web Matter style guide...